MyWeightTests/SnapshotService.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  SnapshotService.swift
+//  MyWeight
+//
+//  Created by Bruno Mazzo on 11/5/16.
+//  Copyright © 2016 Diogo Tridapalli. All rights reserved.
+//
+
+import Foundation
+import Nimble
+import Nimble_Snapshots
+
+public struct SnapshotService {
+    let iPhonePortraitSizes = ["iPhone 4'" : CGSize(width: 320, height: 568),
+                       "iPhone 4.7'" : CGSize(width: 375, height: 667),
+                       "iPhone 5.5'" : CGSize(width: 414, height: 736)]
+
+    public struct ScreenSnapshot {
```

Comment:
About the function names is ok. The problem is that the Nimble-Snapshot already defines a `Snapshot` structure. My structure is a wrapper to allow to pass the sizes to test many in a row. I can change the name and use `SnapshopService.Snapshot` to help the compiler, but I think it can become confusing. What do you think?